Woof/Woof/Model/Sitter/Sitter+Dummy.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+extension Sitter {
+    /// Provides dummy data for `Sitter` model.
+    enum Dummy {
+        /// The instance of dummy sitter to use in preview of views.
+        static let emilyDoe = Sitter(
+            id: UUID(),
+            name: "Emily",
+            surname: "Doe",
+            phone: "1234567890",
+            avatarUrl: Bundle.main.url(forResource: "EmilyDoeAvatar", withExtension: "jpg"),
```

Comment:
nit:

I'd recommend to use the `Woof/Woof/Resources/Images/Mock/EmilyDoeAvatar.jpg`, not the `Woof/Woof/Helper/Resources/Images/Mock/EmilyDoeAvatar.jpg`

`Helper` usually contains the sources; `Resources` contains resources :)